packages/web/components/cards/generic-main-card.tsx

```diff
@@ -1,20 +1,17 @@
 import React, { ReactNode } from "react";
 
+import { Card } from "~/components/cards";
 import { IconLink } from "~/components/cards/icon-link";
 
 export const GenericMainCard: React.FC<{
   children: ReactNode;
   title?: string;
   titleIcon?: ReactNode;
   titleIconAction?: string;
-  width?: string;
-}> = ({ children, title, titleIcon, titleIconAction, width = "27" }) => {
+}> = ({ children, title, titleIcon, titleIconAction }) => {
   return (
-    <div
-      className={`relative flex flex-col gap-8 overflow-hidden rounded-[24px] bg-osmoverse-800 px-1 py-1 lg:mx-auto md:mt-mobile-header md:gap-6 md:px-3 md:pt-4 md:pb-4`}
-      style={{ width: `${width}rem` }}
-    >
-      <div className="flex flex-grow flex-col gap-4 overflow-hidden rounded-[24px] bg-osmoverse-800 px-6 pt-8 pb-8 md:px-3 md:pt-4 md:pb-4">
+    <Card>
```

Comment:
Is there a reason to use the additional nesting of divs here? I can't immediately see what type of HTML element is being used by the `Card` component, but it's safe to assume it's more than is needed